test/cpp/lazy/test_lazy_ops.cpp

```diff
@@ -7113,6 +7113,8 @@ TEST_F(LazyOpsTest, TestViewSqueezeAddInPlace) {
   });
 }
 
+// TODO: reenable after LTC rewrite
```

Comment:
👀

Reply:
I changed the computation logic to stop pinging sizes() and directly access the field, but LTC TensorImpl was relying on that, so now I have to rewrite LTC to deal with it. I roughly know how to do it, just needs to be done.

Reply:
hmm.  is this only an issue for LTC without functionalization? what's your plan for timing this?  we don't want to land this if it breaks xla/cerebras but i'm not quite sure what breaks here